metadata-models/src/main/pegasus/com/linkedin/metadata/search/SearchSuggestion.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.metadata.search
+
+/**
+ * The model for the search result
+ */
+record SearchSuggestion {
+
+  /**
+   * The suggestion text for this search query
+   */
+  text: string
+
+  /**
+   * The score for how close this suggestion is to the original search query.
+   * The closer to 1 means it is closer to the original query and 0 is further away.
+   */
+  score: float
+
+  /**
+   * How many matches there are with the suggested text for the given field
+   */
+  frequency: long
```

Comment:
Should this be optional to match the gql definition?

Reply:
i'll leave this as required for now since we're always setting it on the backend when we get results from elastic. I opted to keep `score` and `frequency` as optional on the API for now in case that ever changes just to give us a little wiggle room.